client/styles/definitions.styl

```diff
@@ -23,6 +23,16 @@ uber-orange = #CA3B27
 uber-green = #629A41
 uber-yellow = #ECAB20
 
+// temporal color palette
+temporal-white = #F2F2F2
```

Comment:
I'm assuming you got these from Shawn?

Reply:
yup, i got it from the landing page code

Reply:
Cool